src/vsphere_cpi/lib/cloud/vsphere/resources/resource_pool.rb

```diff
@@ -27,12 +28,24 @@ def mob
         if @cluster_config.resource_pool.nil?
           @mob = @root_resource_pool
         else
-          client = @client
-          @mob = client.cloud_searcher.get_managed_object(
-            Vim::ResourcePool,
-            :root => @root_resource_pool,
-            :name => @cluster_config.resource_pool)
-          logger.debug("Found requested resource pool: #{@mob}")
+          begin
+            resource_pool_path_suffix = @cluster_config.resource_pool
+            datacenter_cluster_path_prefix = "#{@datacenter_name}/host/#{@cluster_config.name}/Resources/"
+            full_inventory_path = datacenter_cluster_path_prefix + resource_pool_path_suffix
+            # Replace all multiple consecutive / with single /
+            full_inventory_path.gsub!(/\/+/, '/').chomp!('/')
```

Comment:
In which case this may be unnecessary.

Reply:
I don't think escape will remove unnecessary `/` characters.
My concern basically is that the user while defining a resource pool can do it in these ways 
 - `vcpi-rp-1-1/vcpi-sub-rp`
 - `/vcpi-rp-1-1/vcpi-sub-rp`
 -  `vcpi-rp-1-1/vcpi-sub-rp/`
 - `/vcpi-rp-1-1/vcpi-sub-rp/`

I want to let user use any of the above.